### DISCUSSION TITLE
New sources

### DISCUSSION THREAD

**Discussion by rizlas** [bb-Ricardo/netbox-sync#183] [General]
Hi Ricardo,
we are currently developing other sources, would you like to establish a point of contact so we can discuss some parts of the code that are currently blocking us? Slack/discord/email would be nice.

Thank you

**Comment by bb-Ricardo** [comment]
Hey, that sounds great. The email address should be in the help text message or the git commit. Just contact me and we can continue from there.

**Comment by rizlas** [comment]
Unfortunately I cant find it.

**Reply by bb-Ricardo** [reply]
https://github.com/bb-Ricardo/netbox-sync/blob/d0143119de99fd40e39e923fe111045a85796b96/netbox-sync.py#L30

**Reply by rizlas** [reply]
Uh, thanks 📧

**Reply by bb-Ricardo** [reply]
did you send a email? I never got one.

**Reply by rizlas** [reply]
Not yet, we are planning to do it.

**Reply by bb-Ricardo** [reply]
ahh ok, was just wondering if it got lost on the way.
